README.md

```diff
@@ -1,104 +1,130 @@
 ```
 
-## Development
+Notice that we have also provided a `porterUri`. **Porter is a web-based service that interacts with nodes on the network on behalf of applications – an "Infura for TAC".** In this example, we've chosen a `lynx` (testnet) Porter endpoint.
 
-Install git hooks
+## 3. Create Conditions
 
-```bash
-npx husky install
-```
+We will now specify the conditions on which data access will be predicated – i.e. what will the data requester need to prove in order to gain decryption rights. In this tutorial, nodes will check that the requester owns a specific ERC721 NFT:
 
-Generate contract typings
+```javascript
+import { Conditions } from '@nucypher/nucypher-ts';
 
-```bash
-yarn typechain
+const NFTOwnership = new Conditions.ERC721Ownership({
+  contractAddress: '0xBC4CA0EdA7647A8aB7C2061c2E118A18a936f13D',
+  chain: 5, // lynx uses the Goerli testnet
+  parameters: [5954],
+});
 ```
 
-Prepare a new release
+Note that there are other Condition [types](https://docs.threshold.network/app-development/threshold-access-control-tac/references/conditions), and it is possible to compose and combine multiple Condition objects into a [_ConditionSet_](https://docs.threshold.network/app-development/threshold-access-control-tac/references/condition-set)_:_
+
+```javascript
+import { Conditions, ConditionSet } from '@nucypher/nucypher-ts';
 
-```bash
-yarn run prepare-release
+const conditions = new ConditionSet([NFTOwnership]);
 ```
 
-## Publishing
+In this tutorial, we'll only specify a single Condition to access the data.
 
-Publish a new release on NPM.
+## 5. Build a Strategy
 
-Pay attention to output of these commands and fix your release if needed.
+We now bundle the Cohort, ConditionSet, and any other extra parameters into a [_Strategy_](https://docs.threshold.network/app-development/threshold-access-control-tac/references/strategy)_:_
 
-To build and publish a release, run
+```javascript
+import { Strategy } from '@nucypher/nucypher-ts';
 
-```bash
-yarn prepare-release
-# Or, to publish an alpha release
-yarn prepare-release:alpha
+const newStrategy = Strategy.create(
+  newCohort,
+  conditions
+);
 ```
 
-Follow instructions from the command output to finalize the process.
-
-## Examples
+Next, we deploy this Strategy to the Threshold Network.
+In order to make this more affordable, deployments are handled by a L2, in this case we use Polygon Mumbai:
 
-See `./test` directory for usage examples.
+```typescript
+import detectEthereumProvider from '@metamask/detect-provider';
+import providers from 'ethers';
 
-See `./examples` directory for examples of browser integration.
+const MMprovider = await detectEthereumProvider();
+const mumbai = providers.providers.getNetwork(80001);
 
-See `./examples/api-example.ts` for an abridged API example.
+if (MMprovider) {
+  const web3Provider = new providers.providers.Web3Provider(
+    MMprovider,
+    mumbai
+  );
+  const newDeployed = await newStrategy.deploy('test', web3Provider);
```

Comment:
Perhaps we should clarify what the value 'test' is used for?

Reply:
this is an artefact of building ontop of PRE, so it's not actually used for anything. Do we think it will be used with DKG?

Reply:
🤔 (cc @KPrasch , @cygnusv )